### PR TITLE
Make save_model support non-fault-tolerant paddle cloud training

### PR DESCRIPTION
**Pull request by helinwang**
Fixes: https://github.com/PaddlePaddle/Paddle/issues/3625

### CODE REVIEW

File: python/paddle/v2/model.py [hunk 1]
```diff
@@ -33,37 +30,38 @@ def mkdir_p(path):
...
-        client = master.client("http://" + etcd_ip + ":2379", 5, 0)
-        r = client.request_save_model(trainer_id, 5000)
-        if r == 0:
-            # do not need to save
-            return
-        elif r < 0:
-            # error
-            return
+        trainer_id = os.environ.get(trainer_key)
+        etcd_key = "ETCD_IP"
```
Comment (typhoonzero) [inline python/paddle/v2/model.py]: Curious where to set `ETCD_IP` env?

File: python/paddle/v2/model.py [hunk 2]
```diff
@@ -33,37 +30,38 @@ def mkdir_p(path):
...
-    mkdir_p(path)
-
+            etcd_ip = os.environ.get(etcd_key)
+            client = master.client("http://" + etcd_ip + ":2379", 5, 0)
+            r = client.request_save_model(trainer_id, 5000)
+            if r == 0:
+                # do not need to save
+                return
+            elif r < 0:
+                # error
```
Comment (typhoonzero) [inline python/paddle/v2/model.py]: Log or print to `stderr`?

File: python/paddle/v2/model.py [hunk 3]
```diff
@@ -33,37 +30,38 @@ def mkdir_p(path):
             raise
 
 
-def save_model(parameters, path):
+def save_model(parameters, folder, name="model.tar"):
```
Comment (typhoonzero) [inline python/paddle/v2/model.py]: Add doc string?

File: python/paddle/v2/model.py (hunk 1 above)
Comment (helinwang) [inline python/paddle/v2/model.py]: I think we set it here: https://github.com/PaddlePaddle/cloud/blob/cafa1329209b0682ddc17db074b1042b1cb97f45/docker/paddle_k8s#L64

Comment (typhoonzero) [inline python/paddle/v2/model.py]: Cool, I see.

**Comment by helinwang** [comment]
This PR is out-of-date, closing.

**Pull request closed without merging**